### PR TITLE
Automatically open StreetView in a new browser tab when no Google API Key is provided (invalid key or empty string)

### LINKED ISSUE #164: Streetview Map control: Show StreetView on new browser tab if no google key is provide
Steps:  

1. StreeView map control

Screenshot from 2022-09-01 10-39-45

2. Click on map. A new right browser tab is open with Streetview positioned on map coordinates click  

Screenshot from 2022-09-01 10-42-14  Screenshot from 2022-09-01 10-45-42

### PR DESCRIPTION
**Pull request by volterra79**
close: https://github.com/g3w-suite/g3w-client/issues/164

---

Steps:  

1. StreeView map control

Screenshot from 2022-09-01 10-39-45

2. Click on map. A new right browser tab is open with Streetview positioned on map coordinates click  

Screenshot from 2022-09-01 10-42-14  Screenshot from 2022-09-01 10-45-42

---

**Notable changes:**

- remove src/app/gui/streetview/streetviewservice.js (replaced by `src/app/g3w-ol/controls/streetviewcontrol.js`)
- remove unusued `StreetViewService` imports

### CODE REVIEW

File: src/app/gui/map/mapservice.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ const WFSProvider = require('core/layers/providers/wfsprovider');
 const olhelpers = require('g3w-ol/g3w.ol').helpers;
 const {getScaleFromResolution, getResolutionFromScale} = require('core/utils/ol');
 const ControlsFactory = require('gui/map/control/factory');
-const StreetViewService = require('gui/streetview/streetviewservice');
 const ControlsRegistry = require('gui/map/control/registry');
```
Comment (Raruto) [inline src/app/gui/map/mapservice.js:21]: @volterra79 can you confirm that it was only a typo? (i.e. unusued variable)

File: src/app/g3w-ol/controls/streetviewcontrol.js [hunk 2]
```diff
@@ -1,8 +1,69 @@
...
+      }
+      $script(`${GoogleStreetViewApiUrl}js?${this.key ? 'key=' + this.key : '' }`,
+        resolve
+      )
+    })
+  };
+
+  this.getKey = function(){
+    return this.key;
+  };
```
Comment (Raruto) [inline src/app/g3w-ol/controls/streetviewcontrol.js]: @volterra79 can we get rid of?

File: src/app/g3w-ol/controls/streetviewcontrol.js [hunk 3]
```diff
@@ -1,8 +1,69 @@
...
+
+  this.setKey = function(key) {
+    this.key = key;
+  };
```
Comment (Raruto) [inline src/app/g3w-ol/controls/streetviewcontrol.js]: @volterra79 can we get rid of?

File: src/app/g3w-ol/controls/streetviewcontrol.js (hunk 2 above)
Comment (volterra79) [reply src/app/g3w-ol/controls/streetviewcontrol.js]: Can remove it. It is not used

File: src/app/g3w-ol/controls/streetviewcontrol.js (hunk 3 above)
Comment (volterra79) [reply src/app/g3w-ol/controls/streetviewcontrol.js]: Can remove it. Not used

File: src/app/gui/map/mapservice.js (hunk 1 above)
Comment (volterra79) [reply src/app/gui/map/mapservice.js:21]: Yes. No more used

File: src/app/g3w-ol/controls/streetviewcontrol.js [hunk 4]
```diff
@@ -1,12 +1,74 @@
...
+      title: 'StreetView'
+    });
+  };
+
+  base(this);
+}
+
+inherit(StreetviewService, G3WObject);
+
+const streetviewService = new StreetviewService();
```
Comment (Raruto) [inline src/app/g3w-ol/controls/streetviewcontrol.js]: @volterra79  can you include these features within a single class? (ie. `StreetviewService` --> `StreetViewControl`)

Comment (volterra79) [reply src/app/g3w-ol/controls/streetviewcontrol.js]: Sure.

**Comment by Raruto** [comment]
@volterra79 please stop merging stuff related to milestone [v3.8](https://github.com/g3w-suite/g3w-client/milestone/8)

**Comment by volterra79** [comment]
@Raruto is related to milestone v3.7

**Pull request merged**